app/state_machines/form_state_machine.rb

```diff
@@ -0,0 +1,29 @@
+module FormStateMachine
+  extend ActiveSupport::Concern
+
+  included do
+    include AASM
+
+    enum :state, {
+      draft: "draft",
+      live: "live",
```

Comment:
```suggestion
      initial_draft: "initial_draft",
      live: "live",
      draft: "draft"
```

I was wondering if would be be better than just draft/live because we can track how many forms are yet to be made live at all.

```
Form.initial_draft # returns all Forms that have never been made live

Form.live # returns all Forms that are made live

Form.draft # returns Forms that have changes since the form went live and are waiting for the changes to be made live
```

Reply:
* new
* live
* draft
?

Reply:
resolving this as the state names have changed